DigitalLearningSolutions.Data/Services/ImageResizeService.cs

```diff
@@ -67,8 +88,25 @@ private Image CropImageToCentredSquare(Image image)
 
         private Image ResizeSquareImage(Image image, int sideLengthPx)
         {
-            var destRect = new Rectangle(0, 0, sideLengthPx, sideLengthPx);
-            var returnImage = new Bitmap(sideLengthPx, sideLengthPx);
+            return ResizeImageToDimensions(image, sideLengthPx, sideLengthPx);
+        }
+
+        private Image ResizeImageByMaxSideLength(Image image, int maxSideLengthPx)
+        {
+            var ratioX = Math.Min((float)maxSideLengthPx / (float)image.Width, 1);
+            var ratioY = Math.Min((float)maxSideLengthPx / (float)image.Height, 1);
```

Comment:
This ensures that we don't accidentally enlarge a small logo or signature - I thought that would be the expected behaviour, happy to be corrected though

Reply:
That sounds reasonable to me! 👍